articles/getting-started-with-browser-sync/index.md

```diff
@@ -0,0 +1,161 @@
+
+---
+
+layout: engineering-education
+
+status: review
+
+published: false
+
+url: /engineering-education/getting-started-with-browser-sync/
+
+title: Getting Started with Browser-sync
+
+description: The basic concept of browser-sync, installing browser-sync and using it serve your files while building a website project
+
+author: aransiola-ayodele-leom
+
+date: 2020-09-24T00:00:27-21
+
+topic: [Browser-sync]
+
+excerpt_separator: <!--more-->
+
+images:
+
+- url: /engineering-education/getting-started-with-browser-sync/hero.jpg
+
+alt: Computers in a workspace displaying code
+
+---
+<<<<<<< HEAD
```

Comment:
You've committed a merge conflict. See this article from [Github](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-merge-conflicts)

Reply:
I have made a new Pull Request as you have suggested [here](https://github.com/section-io/engineering-education/pull/409
)